packages/vue-dot/src/patterns/FormLayout/layouts/LayoutQuestion/LayoutQuestion.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+	<VLayout
+		column
+		class="vd-layout ma-n4"
+	>
+		<div
+			v-if="getField(0).title"
+			class="mx-4 mt-4"
+		>
+			<h4 class="body-1 my-0">
+				<span>{{ getField(0).title }}</span>
+				<VTooltip right>
+					<template v-slot:activator="{ on }">
+						<VBtn
+							icon
+							v-on="on"
+						>
+							<VIcon>
+								{{ informationIcon }}
+							</VIcon>
+						</VBtn>
+					</template>
+					<span v-html="getField(0).tooltip" />
+				</VTooltip>
+			</h4>
+		</div>
+		<p class="body-2 mx-4 mb-0">
+			{{ getField(0).description }}
+		</p>
+		<slot
+			name="content-1"
+			v-bind="getField(0)"
```

Comment:
Attention, ici il faut modifier aussi !
```suggestion
			:field="getField(0)"
```